src/gobexport/exporter/utils.py

```diff
@@ -65,9 +65,14 @@ def _evaluate_format_action(entity: dict, action: dict):
 
     value = get_entity_value(entity, action['value'])
 
-    if value:
-        return action['formatter'](value)
-    return None
+    if not value:
+        return
+
+    format = action.get('format')
```

Comment:
Is it an option to pass all extra keys (besides formatter, value and action) as kwargs? Format is at this moment very specific to that one timestamp formatter. I think it is a better option to pass everything else as kwargs to stay flexible.

Reply:
Good point. Will do.